CMakeLists.txt

```diff
@@ -65,8 +60,8 @@ include(sdk/cmake/compilerflags.cmake)
 add_definitions(-D__REACTOS__)
 
 # Double escape, since CMake unescapes before putting it on the command-line, where it's unescaped again by GCC/CL.
-add_definitions(-DREACTOS_SOURCE_DIR="\\\"${REACTOS_SOURCE_DIR}\\\"")
-add_definitions(-DREACTOS_BINARY_DIR="\\\"${REACTOS_BINARY_DIR}\\\"")
+add_compile_flags(-DREACTOS_SOURCE_DIR="\\\"${REACTOS_SOURCE_DIR}\\\"")
+add_compile_flags(-DREACTOS_BINARY_DIR="\\\"${REACTOS_BINARY_DIR}\\\"")
```

Comment:
Is it possible to keep these as `add_definitions` and remove the escaping instead? Not sure how either option will deal with the backslashes in Windows paths.

Reply:
Can change it to add_definitions, i had trouble with _RELFILE_ before giving up and making them all compile flags